docs/content/direct/binding.md

```diff
@@ -1,5 +1,179 @@
 # Binding workload with WEC
 
-This document is about BindingPolicy and Binding objects and the concept of downsync.
+This document is about associating WECs with workload objects. The
+primary concept is sometimes called "downsync", which confusingly
+refers to both the propagation and [transformation](transforming.md)
+of desired state from core to WECs _and_ the propagation and
+summarization of reported state from WECs to core.
 
-TODO: write it.
+## Binding Basics
+
+The user controls downsync primarily through API objects of kinds
+`BindingPolicy` and `Binding`. These go in a WDS and associate
+workload objects in that WDS with WECs, along with adding some
+modulations on how downsync is done.
+
+`BindingPolicy` is a higher level concept than `Binding`. KubeStellar
+has a controller that translates each `BindingPolicy` to a
+`Binding`. A user _could_ eschew the `BindingPolicy` and directly
+maintain a `Binding` object or let a different controller maintain the
+`Binding` object (TODO: check that this is true). The `Binding` object
+shows which workload objects and which WECs matched the predicates in
+the `BindingPolicy` and so is also useful as feedback to the user
+about that.
+
+## BindingPolicy
+
+The `spec` of a `BindingPolicy` has two predicates that (1) identify a
+subset of the WECs in the inventory of the ITS associated with the WDS
+and (2) identify a subset of the workload objects in the WDS. The
+primary function of the `BindingPolicy` is to assert the desired
+association between (1) and (2). A `BindingPolicy` can also add some
+modulations on how those workload objects are downsynced to/from those
+WECs.
+
+The WEC-selecting predicate is an array of label selectors in
+`spec.clusterSelectors`. These label selectors test the labels of the
+inventory objects describing the WECs. The bound WECs are the ones
+whose inventory object passes at least one of the the label selectors
+in `spec.clusterSelectors`.
+
+The workload object selection predicate is in `spec.downsync`, which
+holds a list of `DownsyncPolicyClause`s; each includes both a workload
+object selection predicate and also two kinds of information that
+modulate the downsync.
+
+The workload object selection part of a `DownsyncPolicyClause`
+consists of the following fields, all of which are optional. A
+workload object matches a `DownsyncPolicyClause` if that object
+matches all of these selection fields; the default value for each
+field specifies a test that every object passes. Note, however, that
+not every API object in a WDS is considered to be a _workload_
+object. The KubeStellar control objects and `CombinedStatus` objects
+are not considered to be workload objects. (TODO: also talk about the
+other objects that never match).
+
```

Comment:
Instead of all the info in the sections bellow this point - I don't see why we can't just point to the object itself, and add all the needed info as comments there?
And maybe maintain just the yaml example.   Seems like a duplication of the code

Reply:
I have pushed such a revision.